<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminpage</class>
 <widget class="QWidget" name="adminpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="bar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>管理员系统</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="bar_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置系统基础数据</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="bar_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>180</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>配置角色操作权限</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="setTable">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置餐位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addUser">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>260</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>增加角色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setDish">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>345</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置菜品</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setDrink">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>430</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置酒水</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteUser">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>345</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删除角色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="password">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>430</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改角色密码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
